recipes/chunky3d/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "chunky3d" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 768037c8b24977e898a884564cc1447c4a125ba058ffa048e78ab0ca7c9044a5
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
```

Comment:
you don't need this build section at all

Reply:
thanks, fixed.